docs/user/README.md

```diff
@@ -38,25 +38,29 @@ file a bug in [Bugzilla in Data Platform & Tools :: Glean.js][gleanjs-bugs].
 > if you want to use Glean.js and is missing some key Glean feature.
 ## Sections
 
-### [Using Glean](./user/index.html)
+### [User's Guides](./user/adding-glean-to-your-project.html)
+
+This section of the book contains mostly step-by-step guides and essays detailing how to
+achieve specific tasks with Glean.
+
+It contains guides on the first steps of integrating Glean into your project,
+choosing the right metric type for you, debugging products that use Glean and
+Glean's built-in error reporting mechanism.
 
-In this section we describe how to use Glean in your own libraries and applications.
-It explains the first steps of integrating Glean into your project, choosing the right metric type for you,
-debugging products that use Glean and Glean's built-in error reporting mechanism.
 If you want to start using Glean to report data, this is the section you should read.
 
-### [Metric Types](./user/metrics/index.html)
+### [API Reference](./reference/yaml/index.html)
+
+This section of the book contains reference pages for Glean’s user facing APIs.
+
+If you are looking for information a specific Glean API, this is the section you should check out.
 
-This sections lists all the metric types provided by Glean, with examples on how to define them
-and record data using them. Before diving into Glean's metric types details, don't forget to
-read the [Choosing a metric type](https://mozilla.github.io/glean/book/user/adding-new-metrics.html#choosing-a-metric-type) page.
+### [Language Binding Specific Information](./language-bindings/android/index.html)
```

Comment:
In the ToC it says `Implementation Specific Information`. Which do we prefer?

I think `Language Binding Specific Information` is more correct (we use "implementation" to distinguish between SDK and .js above), but I'd be cool with using "Implementation" to refer to LBs across both clients and instead clarifying the above language in the front matter to be "client"

Reply:
Yeah, from the start I did always dislike the "Implementation Specific..." terminology. I made the change based on Alessio's review comment https://github.com/mozilla/glean/pull/1601#discussion_r623020279

Also the TOC was a mistake which I fixed already :D See 2960ff1

It is a fuzzy concept, what is a language binding and what is an implementation of Glean. Definitely something to add to the glossary... I would be fine with calling Glean.js a language binding. [Wikipedia tells me that is not quite correct though](https://en.wikipedia.org/wiki/Language_binding)... Why is terminology so hard?